normalizer/messages.go

```diff
@@ -0,0 +1,61 @@
+// Package writer provides message writer concept definitions.
+package normalizer
+
+import (
+	"github.com/cisco/senml"
+	"github.com/mainflux/mainflux/writer"
+	"go.uber.org/zap"
+)
+
+// Message represents a message emitted by the mainflux adapters layer.
+type MainfluxMessage struct {
```

Comment:
Please avoid prefixing domain objects with `Mainflux`. It does not bring any value except code clutter.

Reply:
Also, note that this change will require a `writer` and `http` update as well.

Reply:
Writer has been updated already, I'll update http.

Any propositions for message struct name? `AdapterMessage`?